### ISSUE TITLE
Why is the ContentType header set to the file name?

### ISSUE THREAD

**Issue by yazinsai** [MathieuLoutre/grunt-aws-s3#54]
I've got the following grunt file:

```js
grunt.initConfig({
    aws: grunt.file.readJSON('aws-credentials.json'), // Read the file
    pkg: grunt.file.readJSON('package.json'),
    aws_s3: {
      options: {
        accessKeyId: '<%= aws.AWSAccessKeyId %>',
        secretAccessKey: '<%= aws.AWSSecretKey %>',
        region: 'eu-west-1',
      },
      clean_production: {
        options: {
          bucket: 'example.com',
        },
        files: [
          {dest: '/', action: 'delete'},
          ],
      },
      production: {
        options: {
          bucket: 'example.com',
          params: {
            ContentEncoding: 'gzip', // applies to all the files!
          }
        },
        files: [
          {expand: true, cwd: 'build/', src: ['**'], dest: ''},
          ],
      }
    },
```
Grunt works fine .. but for some reason, when I inspect the metadata on my S3 panel after the push, I see the file path there instead of the correct `ContentType`:

![image](https://cloud.githubusercontent.com/assets/1846034/5691359/ef4324f8-98db-11e4-94c8-2b66327b4394.png)

I've got `"grunt-aws-s3": "^0.10.1",` in my `package.json`

**Comment by yazinsai** [comment]
Note: Switching to version `0.10.0` resolves the issue.. Seems to be something introduced recently.

**Comment by MathieuLoutre** [comment]
Hi there! The mime type library has changed between 0.10.0 and 0.10.1 which is certainly linked. Would need to take a better look. Thanks for reporting.

**Comment by MathieuLoutre** [comment]
@Takeno just submitted a PR that likely fixes it. I published `0.10.2` if you can try it and let me know if it works for you.

**Comment by MathieuLoutre** [comment]
I had to correct an edge case, so it's finally published in `0.10.3`. Let me know!

**Comment by yazinsai** [comment]
Works now, thanks!
